workers/questionreviewer/server/server.go

```
@@ -0,0 +1,57 @@
+		return
+	}
+
+	ctx, cancel := context.WithTimeout(request.Context(), 10*time.Second)
+	defer cancel()
+	if err := usecase.ReviewQuestion(ctx, server.QuestionRepo, server.ReviewRepo, wrapper.Payload); err != nil {
+		http.Error(writer, "Review failed: "+err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	writer.WriteHeader(http.StatusOK)
+	writer.Write([]byte("Review processed successfully"))
```

golangci-lint (1.64.8)

Error return value of `writer.Write` is not checked

(errcheck)